### PR TITLE
Refactor child components to functional ones 

### PR DESCRIPTION
**Pull request by ChaabaneAnas**
## CHANGES
- refactor `keyboard` component to a functional one
- refactor `display` component to a functional one
refactor `operations` component to a functional one

### CODE REVIEW

File: src/Components/calculator.js [hunk 1]
```diff
@@ -3,58 +3,45 @@
 /* eslint-disable react/prop-types */
 /* eslint-disable jsx-a11y/click-events-have-key-events */
 /* eslint-disable jsx-a11y/no-static-element-interactions */
-/* eslint-disable react/prefer-stateless-function */
-/* eslint-disable max-classes-per-file */
+
```
Comment (codepantha) [inline src/Components/calculator.js]: - [ ] Please don't disable the linters. They are there to help you adhere to style guidelines and write better code 😎

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: REQUIRES CHANGES ♻️ 

## Hi @ChaabaneAnas 👋, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights 🎯 
✔️ Descriptive PR
✔️ Used correct GitFlow
✔️ No linter errors

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Components/calculator.js (hunk 1 above)
Comment (ChaabaneAnas) [reply src/Components/calculator.js]: @codepantha I do understand that they exist for a good reason ...
but some of them are not being fixed ...

**Review by mateo951 — APPROVED** [review]
### Hi @ChaabaneAnas, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🐐 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ChaabaneAnas** [comment]
thank you so much

**Pull request merged**